python/src/scipp/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 from ._scipp.core import *
 from ._scipp import __version__
+from . import detail
```

Comment:
Should this even be included in the init? If we just import it directly where needed there is no risk of using it publicly. On the other hand, maybe we *do* want to have it available everywhere?

Reply:
Initially I kept it away from the `__init__`, and then I thought maybe someone wants to use it in their script, if they are handling large Variables?